src/server/rpc/procedures/roboscape/roboscape.js

```diff
@@ -337,11 +445,15 @@ Robot.prototype.sendToClient = function (msgType, content, fields) {
                     text += ' ' + content[fields[i]];
                 }
 
-                const encryptedContent = {
-                    robot: myself.mac_addr,
-                    message: myself.encrypt(text.trim())
-                };
-                socket.sendMessage('robot message', encryptedContent);
+                socket.send({
```

Comment:
is this an intentional change?

Reply:
This must have been updated on the main brach after i cloned a local copy several months ago. I did not make any changes to encrypt. So I think we need to look back at the state of roboscape immediately after the summer camp, which is when I began editing this file.